test/clustertest/main.cpp

```diff
@@ -38,12 +38,6 @@ int main(int argc, char* argv[]) {
     int threads = 1;
     int repeatCount = 1;
 
-    if (args.isSet("-duplicateRequests")) {
```

Comment:
Was this not used anymore?

Reply:
It was only ever used when I was building the mockRequest feature, and has never been used by any automated tests. I figured we'd take it out and clean it up. It's pretty easy to add back if we ever need it again.